paddle/framework/operator.h

```diff
@@ -376,6 +376,11 @@ class ExecutionContext : public InferShapeContext {
 
   platform::Place GetPlace() const { return device_context_.GetPlace(); }
 
+  template <typename PlaceType>
+  platform::Place GetPlace() const {
```

Comment:
As #3902 , maybe we can keep the style and rename the function name to `Place()`, so we can use it as:
```C++
ctx.Place<PlaceType>();
```